# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1384489252 18000
#      Thu Nov 14 23:20:52 2013 -0500
# Node ID 929c95b33eae74094273ce9892d0b86699aa08ad
# Parent  14a5493a78c85f366b4ffe29b12a39b425e1a749
Restore correct preprocessor statememt in lo-specfun.h

* liboctave/numeric/lo-specfun.h: Restore correct preprocessor
conditionals, broken by changeset ebb3ef964372.

diff --git a/liboctave/numeric/lo-specfun.h b/liboctave/numeric/lo-specfun.h
--- a/liboctave/numeric/lo-specfun.h
+++ b/liboctave/numeric/lo-specfun.h
@@ -36,85 +36,85 @@ class ComplexColumnVector;
 class FloatMatrix;
 class FloatComplexMatrix;
 class FloatNDArray;
 class FloatComplexNDArray;
 class FloatRowVector;
 class FloatComplexColumnVector;
 class Range;
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_ACOSH)
 extern OCTAVE_API double acosh (double);
 #endif
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_ASINH)
 extern OCTAVE_API double asinh (double);
 #endif
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_ATANH)
 extern OCTAVE_API double atanh (double);
 #endif
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_ERF)
 extern OCTAVE_API double erf (double);
 #endif
 extern OCTAVE_API Complex erf (const Complex& x);
 extern OCTAVE_API FloatComplex erf (const FloatComplex& x);
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_ERFC)
 extern OCTAVE_API double erfc (double);
 #endif
 extern OCTAVE_API Complex erfc (const Complex& x);
 extern OCTAVE_API FloatComplex erfc (const FloatComplex& x);
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_ACOSHF)
 extern OCTAVE_API float acoshf (float);
 #endif
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_ASINHF)
 extern OCTAVE_API float asinhf (float);
 #endif
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_ATANHF)
 extern OCTAVE_API float atanhf (float);
 #endif
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_ERFF)
 extern OCTAVE_API float erff (float);
 #endif
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_ERFCF)
 extern OCTAVE_API float erfcf (float);
 #endif
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_EXPM1)
 extern OCTAVE_API double expm1 (double x);
 #endif
 extern OCTAVE_API Complex expm1 (const Complex& x);
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_EXPM1F)
 extern OCTAVE_API float expm1f (float x);
 #endif
 extern OCTAVE_API FloatComplex expm1 (const FloatComplex& x);
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_LOG1P)
 extern OCTAVE_API double log1p (double x);
 #endif
 extern OCTAVE_API Complex log1p (const Complex& x);
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_LOG1PF)
 extern OCTAVE_API float log1pf (float x);
 #endif
 extern OCTAVE_API FloatComplex log1p (const FloatComplex& x);
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_CBRT)
 extern OCTAVE_API double cbrt (double x);
 #endif
 
-#if !defined (octave_lo_specfun_h)
+#if !defined (HAVE_CBRTF)
 extern OCTAVE_API float cbrtf (float x);
 #endif
 
 extern OCTAVE_API double xgamma (double x);
 extern OCTAVE_API double xlgamma (double x);
 extern OCTAVE_API Complex rc_lgamma (double x);
 
 extern OCTAVE_API float xgamma (float x);
